test/unit/address-book-controller.js

```diff
@@ -19,35 +19,39 @@ const mockKeyringController = {
 describe('address-book-controller', function () {
   var addressBookController
 
-  beforeEach(function () {
-    addressBookController = new AddressBookController({}, mockKeyringController)
+  beforeEach(async function () {
+    addressBookController = await new AddressBookController({}, mockKeyringController)
```

Comment:
I see you using `await` with `new` in a few places.  Is this a thing? I don't think async construction is a thing.